pkgs/build-support/pkg-config-wrapper/pkg-config-wrapper.sh

```diff
@@ -12,10 +12,12 @@ if [ -z "${NIX_PKG_CONFIG_WRAPPER_FLAGS_SET_@suffixSalt@:-}" ]; then
     source @out@/nix-support/add-flags.sh
 fi
 
+set -- "$@" @addFlags@
+
 if (( ${#role_suffixes[@]} > 0 )); then
     # replace env var with nix-modified one
     PKG_CONFIG_PATH=$PKG_CONFIG_PATH_@suffixSalt@ exec @prog@ "$@"
 else
-    # pkg-config isn't a bonafied dependency so ignore setup hook entirely
+    # pkg-config isn't a bona fide dependency so ignore setup hook entirely
```

Comment:
easier to understand for non-natives without having to look in the dictionary:

```suggestion
    # pkg-config isn't a real dependency so ignore setup hook entirely
```